### PR TITLE
Remove Kotlin syntax in API doc

### PR DESCRIPTION
**Pull request by cacheung**
Remove Kotlin syntax in API doc

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: Documentation/api-reference.md [hunk 1]
```diff
@@ -66,11 +61,6 @@ Consent.getConsents(new AdobeCallback<Map<String, Object>>() {
 
 #### Kotlin
```
Comment (addb) [inline Documentation/api-reference.md:62]: We should group by Syntax and Example

Sytax
java

Example
java
kotlin

Comment (cacheung) [reply Documentation/api-reference.md:62]: but the syntax is for Java, not for Kotlin, right?

Comment (addb) [reply Documentation/api-reference.md:62]: yes

Comment (cacheung) [reply Documentation/api-reference.md:62]: Let follow below syntax which is the same for adobe io.
Java
Syntax
Example

Kotlin
Example

**Review by addb — APPROVED** [review]

**Pull request merged**